pegasus/sites.v3/code.org/public/help.md

```diff
@@ -92,7 +92,7 @@ Volunteer to teach the Hour of Code or be a guest speaker in a local classroom.
 ### Volunteer weekly with these great organizations
 [col-33]
 
-<img src="/images/fit-300/teals/quincy.jpg" style="border-radius: 5px; margin-left: 0px;">
+<img src="/images/fit-300/teals/quincy.jpg" style="border-radius: 5px; margin-left: 0px;" alt="">
```

Comment:
Just a question here - is this the convention to use for any images that are decorative? To set alt="" Are there any alternatives or benefits to describing the photo? Thank you!

Reply:
I just learned about this this AM! Apparently if you don't have alt text at all, a screen reader will just read the image filename (eeeek) so it's good practice to add an empty alt text so it can skip over entirely in that case. As for when an image should have at text vs not, it seems like if the image 'adds context', then it should be described, but that otherwise it isn't necessary. Erin I hope it's okay I jumped in! I was just very excited about my newfound knowledge (and please correct me/add anything I missed)!

Reply:
Yeah, Hannah is spot-on! Using empty alt reduces audible clutter in screenreaders. Sometimes I find it hard to figure out when an image is decorative, so I really like using this resource: https://www.w3.org/WAI/tutorials/images/decorative. For example, I think both of these images are "Example 4 - image used for ambiance".